packages/story-editor/src/components/library/panes/text/karma/textSets.cuj.karma.js

```diff
@@ -163,6 +163,8 @@ describe('CUJ: Text Sets (Text and Shape Combinations): Using Text Sets', () =>
     // TODO(#9224): Fix flaky test.
     // eslint-disable-next-line jasmine/no-disabled-tests
     xit('should add text color based on background', async () => {
+      fixture.editor.library.text.smartColorToggle.click();
+
```

Comment:
Made a few changes to this test, but couldn't get it to work - not even locally. Not sure what the problem is, but keeping my changes here as they definitely still seem necessary.

Reply:
I guess this PR is partial for #9224 then and fixes #9382, not vice versa?

Would you mind creating an issue for this text sets test specifically? I'll try to debug it a bit as well (probably not before tomorrow)

Reply:
Added an issue: https://github.com/google/web-stories-wp/issues/9633
Switched the issue numbers.